### PR TITLE
Responsive design final

### PR DESCRIPTION
**Pull request by Stephanie041996**
updated media queries
updated nav for mobile view
updated sections for mobile view

### CODE REVIEW

**Comment by steph199607** [comment]
good work

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @Stephanie041996  and @Solenka1.
Very nicely done. This is the only beginning of learning, keep the spirit always up. 🙌 

## The highlights: 

* The git-flow is correct.💯
*  The linters configs are added correctly.💯
*  README is descriptive.👍
*  Nice work with live demo.👌

## What to do next:  ⏩ 
**You need to work some more to get approval for this project**
**Please check my required points below** 👇 
##
**[REQUIRED]**
- [x] You need to create the website for the 3 screen sizes, see from [pathwright](https://microverse.pathwright.com/library/new-technical-curriculum/177956/path/step/104325432/). Currently the tablet size is not working
- [x]  Start with the mobile version and create the media-queries for the tablet and desktop ones. `Tablet: min-width around 768px`, `Desktop: min-width 1024px`.
- [x]  There are a lot of repeated codes in your stylesheets. You can create a custom class for the same code block, like,  .flex {display: flex};
- [x] Create variables for the colors in the stylesheet. Read [here](https://www.w3schools.com/css/css3_variables.asp)
- [x] In your design the `search` icon in Nav3 is not showing, but I can see it in the index file.
- [x] Use the `header` tag for the header, it's the best practice.
- [x] The header sticks to the top of the screen when the user scrolls down. See from the [page](https://perma.cc/M5ZV-Q2D6)
- [x] In desktop design there is a red overlay with a gradient in the top half of the first article.
pic111
Add this gradient design to your project.
- [x] Use the `nav` tag for the `Footer` which contains social media icons centered.
- [x] Add a section in your `README` file to add instructions on how to download and run your project locally. 

##

**Please make the changes and ask for a re-review.**

Feel free to reach me on slack @lamiasristy 🙋
Happy Coding 💻

File: README.md
Comment (whiz25) [inline README.md]: - Give it some space to properly format. `### Steps by download`. This sentence should be something like `### Steps to download`. It makes more sense grammatically.

File: README.md
Comment (whiz25) [inline README.md:41]: - Add space for proper formatting

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice job making the required changes as suggested. Your page looks more like the given design.
Let's make the following changes before merging.
- [x] Add `.vscode` folder to `.gitignore` file.
- [x] Check my comments for some ways to properly format your readme file.


### Martin Afani

**Review by uryelah — APPROVED** [review]
Hi **@Stephanie041996**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job improving your README file and adding `.vscode` to your `.gitignore`.

You should be good to go now. Good luck with the next project! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---
    
## **Optional Suggestions**

- [ ] [OPTIONAL] In order to make sure the `.vscode/` folder, committed by accident, is removed from your repository you first need to follow a couple of steps.

As you may have noticed already, simply including the unrelated folder in the `.gitignore` does not remove it from the repository.

_What you need to do to properly remove the folder is:_
1. Remove it from `.gitignore`
2. Delete the folder/files 
3. Add, commit, and push
4. Re-add it to the `.gitignore`

Then in the next commits you make the folder will not be pushed anymore to your GitHub repo 😀 

_Why should I ignore the .vscode folder?_
That folder is not related to the actual functionality of your project and committing it will bloat it's size and make the repo less tidy. 

---
    
**Keep being an awesome dev and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**
